### PR TITLE
add support xdg base dir

### PR DESCRIPTION
**Pull request by fishioon**

### CODE REVIEW

**Comment by fishioon** [comment]
#32

File: config/config.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func normalizeDir(dir string) (string, error) {
 		return dir, nil
 	}
 
+	if xdg := os.Getenv("XDG_CONFIG_HOME"); xdg != "" {
```
Comment (gsamokovarov) [inline config/config.go]: Can we make sure that we use `~/.jump` if present, so we don't break the existing installs (or force them to move to xdg)? Have you tested that?

Comment (fishioon) [reply config/config.go]: I think it's better to keep ~/.jump if present. I try it.

Comment (gsamokovarov) [reply config/config.go]: Thanks!

**Comment by gsamokovarov** [comment]
I have merged this as is but will iterate a bit on it before shipping it in the next release. Thanks for taking the first step towards XDG support!

**Pull request merged**